solr/contrib/scripting/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version-constraint": "10",
```

Comment:
What is the point of version constraints for local packages?

Can this be removed altogether? If not, then it should be autogenerated by gradle when copying over to the `build/packaging` directory. It should not be hardcoded.

Reply:
Agreed. Local package bug fixes are always released together with Solr, so you'd never want to, say, run Solr 9.0.0 with a LTR package 9.1.0. Makes no sense. So I think we could just remove version-constraint for these manifests, if that it allowed.